php/post-types/class-block-post.php

```diff
@@ -112,11 +112,14 @@ public function register_post_type() {
 			'show_ui'       => true,
 			'show_in_menu'  => true,
 			'menu_position' => 100,
+			// This SVG icon is being included from the plugin directory, so using file_get_contents is okay.
+			// @codingStandardsIgnoreStart
```

Comment:
This is fine, but we need to keep our eyes open for when the WPCS minimum requirements bumps the PHPCS version.  As of PHPCS 3.2+ `@codingStandards*` comments will be replaced with `phpcs:*` comments and eventually dropped entirely in PHPCS 4.0.